cmd/modern/root/config/add-context.go

```diff
@@ -93,4 +94,23 @@ func (c *AddContext) run() {
 		{localizer.Sprintf("To start interactive query session"), "sqlcmd query"},
 		{localizer.Sprintf("To run a query"), "sqlcmd query \"SELECT @@version\""},
 	}, localizer.Sprintf("Current Context '%v'", context.Name))
+	c.LogTelemtry()
+}
+
+func (c *AddContext) LogTelemtry() {
+	eventName := "config-add-context"
```

Comment:
>eventName := "config-add-context" [](http://example.com/codeflow?start=1&length=33)

How does this `eventName` differ from the event name constants defined in internal/telemetry/appinsights.go?